docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_amazon_s3.rst

```diff
@@ -3,16 +3,113 @@
 How to configure an Expectation store in Amazon S3
 ==================================================
 
-.. admonition:: Admonition from Mr. Dickens
+By default, newly profiled Expectations are stored in JSON format in the ``expectations/`` subdirectory of your ``great_expectations/`` folder.  This guide will help you configure Great Expectations to store them in an Amazon S3 bucket.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectations Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Installed `boto3 <https://github.com/boto/boto3>`_ in your local environment.
+    - Identified the S3 bucket and prefix where Expectations will be stored.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+Steps
+-----
+
+1. **Configure** `boto3 <https://github.com/boto/boto3>`_ **to connect to the Amazon S3 bucket where Expectations will be stored.**
+
+2. **Identify your Data Context Expectations Store**
+
+    In your ``great_expectations.yml`` , look for the following lines.  The configuration tells Great Expectations to look for Expectations in a store called ``expectations_store``. The ``base_directory`` for ``expectations_store`` is set to ``expectations/`` by default.
```

Comment:
I re-added the mention of the great_expectations.yml file

Reply:
+1